insights/parsers/dmsetup.py

```diff
@@ -190,7 +190,11 @@ class DmsetupStatus(CommandParser, list):
     def parse_content(self, content):
         self.unparseable_lines = []
         for line in content:
```

Comment:
@ryan-blakley since the content would only have "No devices found" would it make sense to check explicitly for that string and return if found?  That would leave the list empty, and the `unparsable_lines` list empty as well.  Seems like that would reflect the output more accurately.

Reply:
@bfahr I updated it to return if no devices found is in the content, and updated the test to check for empty list.